cmd/server.go

```diff
@@ -53,6 +53,7 @@ const (
 	DataDirFlag                = "data-dir"
 	DefaultTFVersionFlag       = "default-tf-version"
 	DisableApplyAllFlag        = "disable-apply-all"
+	DisableAutoPlanFlag        = "disable-auto-plan"
```

Comment:
We use `autoplan` (one word) for the config so lets use it as one word everywhere.

Reply:
Fixed